tests/util/caches/test_descriptors.py

```diff
@@ -228,8 +228,9 @@ class Cls:
             call_count = 0
 
             @cached()
-            def fn(self, arg1: int) -> Optional[Deferred]:
+            def fn(self, arg1: int) -> Deferred:
```

Comment:
I was gonna say

> Can we specify what value this Deferred resolves to, e.g. `Deferred[None]`?

but it never resolves to a value in this test; it always fails. So ignore this!